extension-manifest-v3/scripts/build.js

```diff
@@ -67,6 +67,14 @@ if (argv.target === 'safari') {
   options.assets.push('background/safari');
 }
 
+if (argv.staging) {
+  execSync('npm run download.staging', { stdio: 'inherit' });
+
+  manifest.debug = true;
+} else {
+  execSync('npm run download', { stdio: 'inherit' });
```

Comment:
Run this as an implicit download function - not an external command.

Reply:
Please add an option to skip download or enable it for build (I think the latter is better)